### PR TITLE
[Docs] Added batch predict tutorial

### PR DESCRIPTION
**Pull request by nschenone**
Added batch predict tutorial using new `batch_predict` function from marketplace

replaced by #2429

### CODE REVIEW

File: docs/tutorial/batch_predict/batch_predict.ipynb [hunk 1]
```diff
@@ -0,0 +1,1042 @@
+{
```
Comment (gilad-shaham) [inline docs/tutorial/batch_predict/batch_predict.ipynb:1]: <p>I don't think we should install mlrun on every demo</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/2415/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (gilad-shaham) [inline docs/tutorial/batch_predict/batch_predict.ipynb:1]: <p>Project name should be different</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/2415/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (gilad-shaham) [inline docs/tutorial/batch_predict/batch_predict.ipynb:1]: <p>Is there a way to publish this, so we won't use development?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/2415/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (nschenone) [reply docs/tutorial/batch_predict/batch_predict.ipynb:1]: <p>In progress</p>

Comment (nschenone) [reply docs/tutorial/batch_predict/batch_predict.ipynb:1]: <p>Recommend taking this up with haviv :)</p>

**Pull request closed without merging**